src/app/product-form/product-form.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+import { ProductItem } from 'src/ProductItem';
+import { ProductService } from '../Services/productService';
+
+@Component({
+  selector: 'app-product-form',
+  templateUrl: './product-form.component.html',
+  styleUrls: ['./product-form.component.css']
+})
+export class ProductFormComponent implements OnInit {
+
+  constructor(public productService:ProductService) { }
```

Comment:
**suggestion:** you should keep services or other injectable classes encapsulation type to private so you don't use them inside the .html files